blacken_docs.py

```diff
@@ -178,6 +184,15 @@ def _latex_match(match: Match[str]) -> str:
         code = textwrap.indent(code, match['indent'])
         return f'{match["before"]}{code}{match["after"]}'
 
+    def _orgmode_match(match: Match[str]) -> str:
+        code = textwrap.dedent(match['code'])
+        code = re.sub(r'( *<<)', r'#\1', code)
```

Comment:
this won't work, for example `x << 5` is valid python code

Reply:
I missed that one completely. I'll upload a repair in a minute. The repair pass my tests on y<<2 and x>>2.